src/main/java/seedu/address/model/client/Client.java

```diff
@@ -0,0 +1,146 @@
+package seedu.address.model.client;
+
+import static java.util.Objects.requireNonNull;
+
+
+/**
+ * Represents a Client associated with a project. This is modelled after the AB3 Person.
+ */
+public class Client {
+
+    //Represents the Client's name
+    private ClientName name;
+
+    //Represents the Client's Email
+    private ClientEmail email;
+
+    //Represents the Client's Phone
+    private ClientPhone phone;
+
+    //Represents the Client's unique ID
+    private ClientId id;
+
+    //Represents a Collection of projects that the client is responsible for
+    private ClientProjectList projects;
+
+    //Represents the client type
+    private Type type;
```

Comment:
Is this field necessary? We are not supporting different client types in this milestone.

Reply:
Noted! However, it is here for two reasons:
1. Since we have decided not to have `Client` as a subclass of `Person`, we must have it sufficiently different from `Person`. One, LSP whereby we couldn't make `Client` act as per our design without violating some LSP aspect. Two, a `Client` can be a person or an organization (external), or the employer. `UNSPECIFIED` can be changed to a message constraint like in `Issue` at a later stage. 
2. It gets hard to implement this afterward so the earlier the class structure is finalized the better. I shall make note of this change while implementing `Client` commands of course.